README.md

```diff
@@ -7,23 +7,37 @@ as part of [Desert Bus For Hope](https://desertbus.org).
 A full design doc can be read at [initial-design-doc.pdf](./initial-design-doc.pdf),
 but a brief overview of the components:
 
+#### Shared Components
+
```

Comment:
I would move ngnix here since it is in front of everything

Reply:
I think the current taxonomy is fine, with "shared" literally meaning "the same code is used in multiple services". We have the "Services" section for things that everything uses (nginx, postgres).